managed-databases/document-database/reference-content/differences-ferret-mongo.mdx

```diff
@@ -0,0 +1,29 @@
+---
+meta:
+  title: Differences between FerretDB and MongoDB
+  description: Learn more about the differences between FerretDB and MongoDB
+content:
+  h1: Differences between FerretDB and MongoDB
+  paragraph: Learn more about the differences between FerretDB and MongoDB
+tags: databases managed-databases document mongodb ferretdb
+categories:
+  - managed-databases
+---
+
+FerretDB uses the same protocol as MongoDB, but there are know differences between the two you must take into consideration when using FerretDB.
+
+For example, even though error names and codes used by FerretDB are the same as MongoDB, but the precise content of the error messages may differ in some cases.
```

Comment:
I would add before a message about the difference in terms of features (ie. some mongo features are not implemented, but all the content below mostly relates to features which are implemented, but still have some small differences compared to standard mongodb api when used)
```suggestion
(...) take into consideration when using FerretDB, even for features already supported fully by FerretDB.
```